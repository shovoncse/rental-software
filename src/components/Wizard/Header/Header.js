import React from 'react';
import { PageHeader } from 'antd';
import styles from './Header.module.css';

const Header = ({ title, subTitle }) => {
  return (
    <>
      <PageHeader
        className={styles.header}
        onBack={() => null}
        title={title}
        subTitle={subTitle}
      />
    </>
  );
};

export default Header;
