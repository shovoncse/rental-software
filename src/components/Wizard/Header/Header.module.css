.header {
    border: 1px solid rgb(235, 237, 240);
    margin: 10px;
    background: #53bbf7;
  }