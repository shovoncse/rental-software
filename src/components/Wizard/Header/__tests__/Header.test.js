import React from 'react';
import ReactDOM from 'react-dom';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Header from '../Header';

afterEach(cleanup);

describe('<Header />', () => {
  it('should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Header></Header>, div);
  });

  it('should render an <div> tag', () => {
    const {
      container: { firstChild },
    } = render(<Header />);
    expect(firstChild.tagName).toEqual('DIV');
  });

  it('should have a class attribute', () => {
    const {
      container: { firstChild },
    } = render(<Header />);
    expect(firstChild.hasAttribute('class')).toBe(true);
  });

  it('matches snapshot', () => {
    const tree = renderer.create(<Header></Header>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
