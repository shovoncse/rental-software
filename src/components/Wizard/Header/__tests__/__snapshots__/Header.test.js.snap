// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> matches snapshot 1`] = `
<div
  className="ant-page-header header ant-page-header-ghost"
/>
`;
